config_generation/db_to_xml.py

```diff
@@ -148,35 +148,265 @@ def convert_template_to_scraper(self, collection) -> None:
         scraper_config = self.update_config_xml()
         return scraper_config
 
-    def convert_template_to_plugin_indexer(self, scraper_editor) -> None:
+    def convert_template_to_job(self, collection, job_type) -> None:
         """
-        assuming this class has been instantiated with the scraper_template.xml
+        assuming this class has been instantiated with the job_template.xml
+        """
+        if job_type == "scrapers":
+            self.update_or_add_element_value("Collection", f"/scrapers/{collection.config_folder}/")
+        elif job_type == "indexer":
+            self.update_or_add_element_value("Collection", f"/SDE/{collection.config_folder}/")
+        job_config = self.update_config_xml()
+        return job_config
```

Comment:
Maybe don't take a type with if statements, but instead just pass through the source name into the f string.